Model/Resolver/CreditMemo/CreditMemoItems.php

```diff
@@ -157,7 +157,7 @@ protected function formatDiscountDetails(
             $discounts[] = [
                 'label' => $associatedOrder->getDiscountDescription() ?? __('Discount'),
                 'amount' => [
-                    'value' => abs($creditmemoItem->getDiscountAmount()) ?? 0,
+                    'value' => abs((int)$creditmemoItem->getDiscountAmount()) ?? 0,
```

Comment:
I believe the type casting will convert null to 0, then we down need the coalesce operator here.
Please check it and if it's true, remove it, please.